docs/en-US/component/statistic.md

```diff
@@ -40,10 +40,10 @@ statistic/card
 ### Statistic Attributes
 
 | Attribute         | Description                    | Type                                                                | Default |
-|-------------------|--------------------------------|---------------------------------------------------------------------|---------|
-| value             | Numerical content              | ^[string] / ^[number]                                               | 0       |
+| ----------------- | ------------------------------ | ------------------------------------------------------------------- | ------- |
+| value             | Numerical content              | ^[number]                                                           | 0       |
 | decimal-separator | Setting the decimal point      | ^[string]                                                           | .       |
-| formatter         | Custom numerical presentation  | ^[Function]`(value: string \| number) => string \| number`          | —       |
+| formatter         | Custom numerical presentation  | ^[Function]`(value: number \| Dayjs) => string \| number`           | —       |
```

Comment:
`value` type should be consistent with `props.value` type

Reply:
updated.